### PR TITLE
completed prep-javascript-objects (Mitchell Maliglig)

### PR DESCRIPTION
**Pull request by MitchellMaliglig**
image

### CODE REVIEW

File: prep-javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+console.log(person);
+
+let fullName = person.firstName + ' ' + person.lastName;
+console.log("The person's name is: " + fullName);
+
+person.job = 'artist';
+console.log("The person's current job is: " + person.job);
+
+person.previousJob = 'pizza boy';
```
Comment (robgmerrill) [inline prep-javascript-objects/main.js]: Bracket notation is requested per the instructions
`person['previousJob'] = 'pizza boy';`

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Good start! Small change requested.

**Comment by MitchellMaliglig** [comment]
image

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**